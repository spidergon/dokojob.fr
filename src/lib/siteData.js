const siteData = {
  title: 'DokoJob',
  description: 'Votre emploi num√©rique en Guyane et en Outre-mer.',
  email: 'contact@dokojob.fr',
  twitter: 'chrisservius',
};

export default siteData;
