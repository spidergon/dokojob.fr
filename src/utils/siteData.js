export default () => ({
  title: 'Jobs',
  description: 'Votre emploi digital en Guyane et en Outre-mer.',
  socials: {
    twitter: 'chrisservius',
    fb: '',
    email: 'hello@example.com',
  },
});
