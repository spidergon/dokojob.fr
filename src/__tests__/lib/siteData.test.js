import siteData from '@lib/siteData';

it('renders lib/siteData unchanged', () => {
  expect(siteData).toMatchSnapshot();
});
