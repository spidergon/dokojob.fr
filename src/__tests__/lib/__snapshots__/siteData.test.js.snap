// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders lib/siteData unchanged 1`] = `
Object {
  "description": "Votre emploi num√©rique en Guyane et en Outre-mer.",
  "email": "contact@dokojob.fr",
  "title": "DokoJob",
  "twitter": "chrisservius",
}
`;
